src/lib.rs

```diff
@@ -80,6 +88,44 @@ impl Write for Vga {
     }
 }
 
+pub static BUFFER: Mutex<Vga> = Mutex::new(
+    Vga {
+        location: 0xb8000 as *mut u8,
+        buffer: [0; ROWS * COLS * 2],
+        position: 0,
+    }
+);
+
+/// Prints something to the screen, with a trailing newline.
+///
+/// # Examples
+///
+/// ```
+/// kprintln!("Hello, world!");
+/// ```
+#[macro_export]
+macro_rules! kprintln {
```

Comment:
1. `util.rs` is a bad name, but I'm not sure if you're married to the idea of that name or it's just to not have it in this file
2. I actually want to kill this entirely, but that requires upstream refactoring first.

so, my vote would be to just leave it in for now.